packages/primitives/src/svg/index.js

```diff
@@ -26,7 +26,9 @@ export const SVG = ( { className, isPressed, ...props } ) => {
 		className:
 			classnames( className, { 'is-pressed': isPressed } ) || undefined,
 		role: 'img',
-		'aria-hidden': 'true',
+		'aria-hidden': props.hasOwnProperty( 'aria-hidden' )
```

Comment:
Did you consider using a new prop like `isFocusable` to make it more explicit?

Reply:
My focus for this PR was only the `aria-hidden` attribute. Nevertheless, if we'd like to make `focusable` attribute optional with default value set, wouldn't it be better if we move it above the props spread instead of creating custom `isFocusable` prop? E.g.

```js
export const SVG = ( { className, isPressed, ...props } ) => (
	<svg
		className={
			classnames( className, { 'is-pressed': isPressed } ) || undefined
		}
		role="img"
		aria-hidden="true"
		focusable="false"
		{ ...props }
	/>
);
```

Reply:
Yes, it would help in this case. I'm not against this change, I just wanted to raise the awareness of the React Native and all the challenges it creates.